spec/lib/object_serializer_spec.rb

```diff
@@ -30,6 +30,30 @@
       expect(serializable_hash[:included]).to be nil
     end
 
+    it 'returns correct nested includes when serializable_hash is called' do
```

Comment:
We should add more tests

- test a has many
- test a has_one
- test a multi level case for example 'actors.agency.adresses'
- we should test this with a polymorphic relationship
- we should probably also test it with a relationship block https://github.com/Netflix/fast_jsonapi/pull/195